x-pack/filebeat/module/suricata/eve/test/eve.json

```diff
@@ -0,0 +1,8 @@
+{"timestamp":"2018-07-05T15:01:09.820360-0400","flow_id":298824096901438,"in_iface":"en0","event_type":"ssh","src_ip":"192.168.86.85","src_port":55406,"dest_ip":"192.30.253.112","dest_port":22,"proto":"TCP","ssh":{"client":{"proto_version":"2.0","software_version":"OpenSSH_7.6"},"server":{"proto_version":"2.0","software_version":"libssh_0.7.0"}}}
```

Comment:
Would be nice to have a `test` and `generated` file as we do in other filesets in addition to this. This would also make it more visible how the end events look like.

The problem is I you would have to copy the module to the module directory and then run the test / generate script there.

Reply:
Ok I see what you mean with example [test.log](https://github.com/elastic/beats/blob/36993290a4d607662684a19e82830db87e875eec/filebeat/module/apache2/access/test/test.log) and [test.log-expected.json](https://github.com/elastic/beats/blob/36993290a4d607662684a19e82830db87e875eec/filebeat/module/apache2/access/test/test.log-expected.json).

These tests run which part of the processing? Currently only a little bit of work is performed in Beats (read as JSON as is to `suricata.eve.*`), and the rest of the work is performed in ingest node. I assume the tests don't run an ES ingest node?

Reply:
The test run ingest node. Have a look at `test_modules.py`